app/views/patients/_pledge.html.erb

```diff
@@ -33,9 +33,9 @@
 
     <div class="hide" data-step="3" data-title="<%= t('patient.pledge.step_three.title') %>">
       <% unless ENV['DARIA_PLEDGE_GENERATOR_DISABLED'] %>
-        <p class="row"><%= t('patient.pledge.step_three.generated', fund: "#{FUND}") %></p>
+        <p class="row"><%= t('patient.pledge.step_three.generated', fund: current_tenant.name) %></p>
         <p class="row">
-          <%= t('patient.pledge.step_three.downloads') %> <%= link_to FUND_FAX_SERVICE, "http://#{FUND_FAX_SERVICE}", target: '_blank', rel: 'noopener nofollow' %>
+          <%= t('patient.pledge.step_three.downloads') %> <%= fax_service %>
```

Comment:
This was the only place we were using this env var constant, andit probably should be the config var regardless.

Reply:
confirmed this was okay